src/bun.js/node/node_fs.zig

```diff
@@ -1136,7 +1136,7 @@ pub const Arguments = struct {
         }
 
         pub fn fromJS(ctx: JSC.C.JSContextRef, arguments: *ArgumentsSlice, exception: JSC.C.ExceptionRef) ?Truncate {
-            const path = PathOrFileDescriptor.fromJS(ctx, arguments, bun.default_allocator, exception) orelse {
+            const path = PathOrFileDescriptor.fromJS(ctx, arguments, bun.default_allocator, .truncate, exception) orelse {
```

Comment:
We can't pass the syscall to the error? We need to pass it to each of these call sites?